darkmoon/api/v1/metadata/views.py

```diff
@@ -101,6 +101,45 @@ async def list_metadata(
         raise ServerNotFoundException(status_code=504, detail="Server timed out.")
 
 
+@router.get(
+    "/listMetadata",
```

Comment:
Since this is the actual base list metadata logic this shouldn't have a different path. The path for the hash lookup should be like `/metadata/hashSearch/` or something like that instead.